docs/includes/quickstart-yo-start-server-excel.md

```diff
@@ -17,11 +17,17 @@ Complete the following steps to start the local web server and sideload your add
     npm start
     ```
 
-- To test your add-in in Excel on a browser, run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running).
+- To test your add-in in Excel on a browser, run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running). Replace "{url}" with the URL of an Excel document on your OneDrive or a SharePoint library to which you have permissions.
 
     ```command&nbsp;line
-    npm run start:web
-    ```
+    npm run start:web -- --document {url}
```

Comment:
I got this command to run the add-in's web service and launch the Excel workbook, but it never sideloaded anything. Is that your experience too?

Reply:
It's not sideloading for me today either. I'll report this as a bug. Also, I need to change the fallback instructions right after the code block to point to the *manual* sideloading section, not the main sideloading section which simply repeats things that the reader has already done by this point.